### PR TITLE
binlog: update documents for committs and passoword

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Part solve https://github.com/pingcap/tidb-binlog/issues/1045
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb-binlog/issues/1045

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@3pointer 目前 2.1/3.0/3.1 是非活跃分支，文档站对应的版本都有这样的提示，是否能考虑不 cp 到这三个分支呢？
image

File: tidb-binlog/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -326,8 +326,8 @@ Drainer="192.168.0.13"
         host = "192.168.0.13"
         user = "root"
         password = ""
-        # 使用 `./binlogctl -cmd encrypt -text string` 加密的密码
-        # encrypted_password 非空时 password 会被忽略
+        # 如果你不想在配置文件中写明文密码，则可以使用 `./binlogctl -cmd encrypt -text string` 生成加密的密码
+        # encrypted_password 非空时 password 会被忽略, encrypted_password 和 password 二选一即可。
```
Comment (IANTHEREAL) [inline tidb-binlog/deploy-tidb-binlog.md:330]: can we put it before L328？

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 2]
```diff
@@ -172,8 +172,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/d
 
 ### initial-commit-ts
 
-* 指定从哪个 commit timestamp 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
-* 默认：`-1`。Drainer 会从 PD 得到一个最新的 timestamp 作为初始时间。
+* 指定从哪个 commit ts 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
```
Comment (IANTHEREAL) [inline tidb-binlog/tidb-binlog-configuration-file.md:175]: can we change `commit ts` to `[PD TSO](link xxx)`

**Comment by IANTHEREAL** [comment]
Rest LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md (hunk 2 above)
Comment (3pointer) [reply tidb-binlog/tidb-binlog-configuration-file.md:175]: Actually, I didn't find the strict definition(link) of `TSO` in our document. 🤔️ we have lots of similar descriptions of this concept in many documents.

File: tidb-binlog/deploy-tidb-binlog.md (hunk 1 above)
Comment (3pointer) [reply tidb-binlog/deploy-tidb-binlog.md:329]: Done

File: tidb-binlog/tidb-binlog-configuration-file.md (hunk 2 above)
Comment (IANTHEREAL) [reply tidb-binlog/tidb-binlog-configuration-file.md:175]:
```suggestion
* 指定从哪个事务提交时间点（事务的 commit ts） 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
```

Comment (IANTHEREAL) [reply tidb-binlog/tidb-binlog-configuration-file.md:175]: LOL

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 3]
```diff
@@ -172,8 +172,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/d
...
+* commit ts 是 TiDB 内部的概念，是从 PD 获取的全局唯一递增的时间戳作为当前事务的唯一 ID，典型 initial-commit-ts 配置可以通过以下方式获得
```
Comment (IANTHEREAL) [inline tidb-binlog/tidb-binlog-configuration-file.md:176]:
```suggestion
* commit ts 是 TiDB 事务的提交时间点，是从 PD 获取的全局唯一递增的时间戳作为当前事务的唯一 ID，典型 initial-commit-ts 配置可以通过以下方式获得
```

Comment (IANTHEREAL) [reply tidb-binlog/tidb-binlog-configuration-file.md:176]: 也可以加上这个 link https://docs.pingcap.com/zh/tidb/stable/transaction-overview

File: tidb-binlog/deploy-tidb-binlog.md [hunk 4]
```diff
@@ -325,9 +325,9 @@ Drainer="192.168.0.13"
         [syncer.to]
         host = "192.168.0.13"
         user = "root"
+        # 如果你不想在配置文件中写明文密码，则可以使用 `./binlogctl -cmd encrypt -text string` 生成加密的密码
+        # encrypted_password 非空时 password 会被忽略, encrypted_password 和 password 二选一即可。
```
Comment (TomShawn) [inline tidb-binlog/deploy-tidb-binlog.md:329]: 是说如果配置了 encrypted_password 且非空，那么配置的 password 不生效吗？

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 5]
```diff
@@ -172,8 +172,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/d
 
 ### initial-commit-ts
 
-* 指定从哪个 commit timestamp 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
-* 默认：`-1`。Drainer 会从 PD 得到一个最新的 timestamp 作为初始时间。
+* 指定从哪个事务提交时间点（事务的 commit ts） 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
+* commit ts 是 TiDB [事务](/transaction-overview.md)的提交时间点，是从 PD 获取的全局唯一递增的时间戳作为当前事务的唯一 ID，典型 initial-commit-ts 配置可以通过以下方式获得
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:176]:
```suggestion
* commit ts (commit timestamp) 是 TiDB [事务](/transaction-overview.md)的提交时间点。该时间点是从 PD 获取的全局唯一递增的时间戳，作为当前事务的唯一 ID。典型 initial-commit-ts 配置可以通过以下方式获得：
```

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 6]
```diff
@@ -172,8 +172,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/d
...
+    - BR 备份元信息 backupmeta 中记录的 backup TS
+    - Dumpling 备份元信息 metadata 中记录的 Pos
+    - PD control 中 tso 命令返回的结果
```
Comment (TomShawn) [inline tidb-binlog/tidb-binlog-configuration-file.md:179]:
```suggestion
    - BR 备份的元信息（即 backupmeta）中记录的 backup TS
    - Dumpling 备份的元信息（即 metadata）中记录的 Pos
    - PD Control 中 `tso` 命令返回的结果
```

File: tidb-binlog/deploy-tidb-binlog.md (hunk 4 above)
Comment (3pointer) [reply tidb-binlog/deploy-tidb-binlog.md:329]: 对

Comment (TomShawn) [reply tidb-binlog/deploy-tidb-binlog.md:329]:
```suggestion
        # 如果配置了 encrypted_password 且非空，那么配置的 password 不生效。encrypted_password 和 password 无法同时生效。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**